gradle.properties

```diff
@@ -26,7 +26,7 @@ intellijPluginVersion=1.3.0
 javassistVersion=3.28.0-GA
 kotlinVersion=1.6.10
 mockitoKotlinVersion=4.0.0
-projectorClientVersion=bc829b35
+projectorClientVersion=a5fb8893df
```

Comment:
Well, I would recommend researching why it happened. Probably it was a temporary problem on JitPack and the build should be rerun there. Please ask for help if it's still unclear